overlord/hookstate/ctlcmd/services_test.go

```diff
@@ -308,8 +316,8 @@ func (s *servicectlSuite) TestQueuedCommands(c *C) {
 	c.Assert(err, IsNil)
 	c.Check(installed, DeepEquals, []string{"one", "two"})
 	c.Assert(tts, HasLen, 2)
-	c.Assert(tts[0].Tasks(), HasLen, 13)
-	c.Assert(tts[1].Tasks(), HasLen, 13)
+	c.Assert(kinds(tts[0].Tasks()), HasLen, 14)
```

Comment:
I'm not sure I understand what is the point of this change.

Reply:
the main one is 13->14; wrapping the Tasks call in kinds() means that the next time it fails, you get to see the kinds instead of a sea of pointers